ember-flight-icons/tests/integration/components/flight-icon-test.js

```diff
@@ -60,6 +60,21 @@ module('Integration | Component | flight-icon', function (hooks) {
       fill: 'rgb(186, 34, 38)',
     });
   });
-  // TODO: the component should have `color` set to `currentColor` by default
+  // the component should have `color` set to `currentColor` (black) by default
+  test('the fill should be set to black by default', async function (assert) {
+    await render(hbs`<FlightIcon @name="meh" />`);
+    assert.dom(`svg.flight-icon`).hasStyle({
+      fill: 'rgb(0, 0, 0)',
+    });
+  });
+  // currentColor should change per inheritance rules
+  test('The fill color should be able to be inherited from parent', async function (assert) {
```

Comment:
thanks for making the tests, this is awesome!

suggestion: consider having the comment and test title be one, for code maintainability and clarity

```suggestion
  test('the component should have `color` set to `currentColor` (black) by default', async function (assert) {
    await render(hbs`<FlightIcon @name="meh" />`);
    assert.dom(`svg.flight-icon`).hasStyle({
      fill: 'rgb(0, 0, 0)',
    });
  });
  test('The fill color (currentColor) should be able to be inherited from parent', async function (assert) {
```

Reply:
@amyrlam do you mind if I put that in a separate PR? I have comments above all of the tests, not just these ones. Happy to remove all of them at once.

Reply:
@MelSumner yes sure, sgtm, thanks!